From 8ea894edf0d761597036ae630600105e302caf5a Mon Sep 17 00:00:00 2001
From: Akira Ajisaka <aajisaka@apache.org>
Date: Wed, 18 Nov 2015 17:23:00 +0900
Subject: [PATCH 0955/1023] HADOOP-12568. Update core-default.xml to describe
 posixGroups support. Contributed by Wei-Chiu
 Chuang.

(cherry picked from commit 4a3b8f1ed20aaa6c9503a2888339a0ec8a6fefc1)

Change-Id: I18c08269fe271db69a1623c97ff7c4a7c978b6da
(cherry picked from commit c98649702259f6f7663ea853be39240bdc4f1530)
---
 .../src/main/resources/core-default.xml            |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
index b938eaa..f2561db 100644
--- a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
+++ b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
@@ -310,6 +310,11 @@ for ldap providers in the same way as above does.
     an LDAP server with a non-AD schema, this should be replaced with
     (&amp;(objectClass=inetOrgPerson)(uid={0}). {0} is a special string used to
     denote where the username fits into the filter.
+
+    If the LDAP server supports posixGroups, Hadoop can enable the feature by
+    setting the value of this property to "posixAccount" and the value of
+    the hadoop.security.group.mapping.ldap.search.filter.group property to
+    "posixGroup".
   </description>
 </property>
 
@@ -319,7 +324,9 @@ for ldap providers in the same way as above does.
   <description>
     An additional filter to use when searching for LDAP groups. This should be
     changed when resolving groups against a non-Active Directory installation.
-    posixGroups are currently not a supported group class.
+
+    See the description of hadoop.security.group.mapping.ldap.search.filter.user
+    to enable posixGroups support.
   </description>
 </property>
 
-- 
1.7.9.5

